### PR TITLE
Implement New Search Bar

### PR DESCRIPTION
**Pull request by p3rc1us**
## Sample
image
## PR Summary
- Changed search bar to prevent turbo frame redundancy #96
- Merge with #95 to resolve Issue #96
- New search bar
- Included select input for Category
- Divided search inputs for What (Service), Where (City), Who (Freelancer)
> Note: Don't forget to `precompile`

### CODE REVIEW

**Review by shinichikudo000 — APPROVED** [review]

File: app/views/services/_search_form.html.erb
Comment (TyJacalan) [inline app/views/services/_search_form.html.erb:1]: looks better but review the mobile responsiveness

**Review by TyJacalan — CHANGES_REQUESTED** [review]
Good to go after reverting the muted color change. Feel free to introduce a new color in the config.

The mobile responsiveness issues can be worked on in a future commit.

File: app/assets/stylesheets/shadcn.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
     --primary-foreground: 26 83.3% 14.1%;
     --secondary: 60 4.8% 95.9%;
     --secondary-foreground: 24 9.8% 10%;
-    --muted: 60 4.8% 95.9%;
```
[the remark this answers is not in the document]
Comment (p3rc1us) [reply app/assets/stylesheets/shadcn.css:13]: @cimafrancamae e68c4d5bc38439878e17b88a2100ab7fa390d022

[the remark this answers is not in the document]
Comment (TyJacalan) [reply app/assets/stylesheets/shadcn.css:13]: I can update the notifications styles if you've already used `muted` extensively. I'm not sure where else I've used it though and some components from shadcn may be using it too.

File: app/views/services/_search_form.html.erb
Comment (p3rc1us) [reply app/views/services/_search_form.html.erb:1]: I'll follow it up. I'm struggling with tw

File: app/assets/stylesheets/shadcn.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (cimafrancamae) [reply app/assets/stylesheets/shadcn.css:13]: It's okay to revert the muted color. Has this been worked out na? @TyJacalan @p3rc1us

**Review by cimafrancamae — APPROVED** [review]

**Review by TyJacalan — APPROVED** [review]

**Pull request merged**